index.yaml

```diff
@@ -82,13 +82,13 @@ distributions:
     - sloretz+buildfarm@openrobotics.org
     release_builds:
       default: noetic/release-build.yaml
-      db: noetic/release-buster-build.yaml
-      dbv8: noetic/release-buster-arm64-build.yaml
+#      db: noetic/release-buster-build.yaml
+#      dbv8: noetic/release-buster-arm64-build.yaml
```

Comment:
My general preference is to delete rather than comment out config lines that are no longer required but this is convention for the Debian decomm process and I think that it does improve discoverability of the fact that Debian Buster was at one point a supported platform. I don't think this needs to change for this one.